From 94a3e8bcac5fc62d04abf4ed2266b3abb8cbfe99 Mon Sep 17 00:00:00 2001
From: Vignesh Viswanathan <quic_viswanat@quicinc.com>
Date: Tue, 4 Jun 2024 19:28:15 +0530
Subject: [PATCH] driver: soc: qcom: Add description for QMI helpers in Kconfig

Add description for QMI helpers in Kconfig so that Kconfig ensures it is
either set or not set based on the driver that selects this config.

Change-Id: I9f69d87c8711c86a9818c91f4dfe1bff9970c78b
Signed-off-by: Vignesh Viswanathan <quic_viswanat@quicinc.com>
---
 drivers/soc/qcom/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/qcom/Kconfig b/drivers/soc/qcom/Kconfig
index 40c38dc9b62b..c70afae5c80f 100644
--- a/drivers/soc/qcom/Kconfig
+++ b/drivers/soc/qcom/Kconfig
@@ -143,7 +143,7 @@ config QCOM_PMIC_GLINK
 	  platforms.
 
 config QCOM_QMI_HELPERS
-	tristate
+	tristate "Qualcomm QMI Helper"
 	depends on NET
 
 config QCOM_RAMP_CTRL
-- 
2.34.1

